lib/internal/test_runner/runner.js

```diff
@@ -32,6 +34,9 @@ const {
 const { basename, join, resolve } = require('path');
 const { once } = require('events');
 
+const kMinPort = 1024;
+const kMaxPort = 65535;
+const kInspectArgRegex = /--inspect(?:-brk|-port)?/;
```

Comment:
Why the RE test instead of checking the option?

Reply:
1. since I copied it from `node:cluster`
2. it makes sense since there are many combinations

Reply:
and now I saw it has a default value so it won't work anyway:
```js 
> ./node --expose-internals -e "console.log(require('internal/options').getOptionValue('--inspect-port'))"
{ host: '127.0.0.1', port: 9229 }
```